src/simple_http.rs

```diff
@@ -362,6 +362,21 @@ impl Builder {
         self
     }
 
+    /// Add proxy address to the transport for SOCKS5 proxy
+    #[cfg(feature = "proxy")]
```

Comment:
This stuff might have slipped into the wrong patch? The "proxy" feature is added in the following patch.

Reply:
My bad.. Parallax error.. Fixed..